Tensile/KernelWriterAssembly.py

```diff
@@ -5842,6 +5927,14 @@ def openSumAtLeastUnroll(self, kernel, prefetch, isPap, isOptNLL):
       kStr += inst("s_cbranch_scc0 %s"%skipOptNLL, \
           "skip if tail loop required")
 
+      # The prefetch across persistent for OptNLL case
+      if self.prefetchAcrossPersistent:
+        kStr += str(self.openPrefetchAcrossPersistent(kernel, isOptNLL=True))
+        newTileCodes = self.setupNewTile(kernel, self.tPA, self.tPB, isPap=True, isOptNLL=True)
```

Comment:
Offset Ptrs for new next PK tile is same as current one. Can we define  another set of SRDA,SRDB for next tile and do all SRDs code after we issue prefetch for the current tile in setupNewTile code?? 

noloadLoop() should be same as unroll loop with global loads & lwr in unroll iteration is for next Tile. 

may be that's next optimization for PK